src/RelayServer.ts

```diff
@@ -548,11 +523,20 @@ export class RelayServer extends EventEmitter {
       envelopingTransaction.metadata.relayMaxNonce.toString()
     );
 
+    await validateIfGasAmountIsAcceptable(envelopingTransaction);
+
+    //FIXME: If the operation is sponsored this calculation is probably useless
```

Comment:
I don't think so, since `maxPossibleGas` is also used to calculate `maxPossibleGasWithViewCall`

Reply:
Done.